routing/routing_tests/turns_tts_text_tests.cpp

```diff
@@ -156,6 +156,77 @@ UNIT_TEST(GetTtsTextTest)
   TEST_EQUAL(getTtsText.GetTurnNotification(notification4), "Затем. Поворот налево.", ());
 }
 
+UNIT_TEST(GetTtsStreetTextTest)
+{
+  string const engShortJson =
```

Comment:
Values should be taken from real files, otherwise what is the point of this test?

Reply:
I didn't create this testing methodology I just used what existed. The point of this test isn't to verify grammar, but to verify the logic of the TTS function. I want to ensure that the correct strings are generated for the provided inputs (for example, no issues with RTL string concatenation, no issues with Unicode regex, no issues with determining when to add the street name vs not add it, etc) the specific strings being used aren't important. If locale json files aren't being parsed we'll know about it pretty quickly, but if Arabic TTS generation suffers a bug we may never know without this test.